### DISCUSSION TITLE
Delete/Remove Vercel Project via REST API

### DISCUSSION THREAD

**Discussion by kashyapro** [vercel/community#3601] [Help] [answered]
### Summary

I am working in a team and have feature for dynamically creating vercel project deployments and its deployed on our vercel team account.

Now I want to delete projects on various scenarios like we no longer need that project running. I can see "Delete Project" option from dashboard but its not feasible for us to delete projects from dashboard manually.

I would love to have some REST API just like we have for creating vercel project. 

### Steps to Reproduce

Not able to find the REST API for delete vercel project.

**Comment by amyegan** [comment, marked as answer]
Hey @kashyapro. You can [delete a project](https://vercel.com/docs/rest-api/endpoints#delete-a-project) via the `/v9/projects/{idOrName}` endpoint. Full endpoint documentation can be found at https://vercel.com/docs/rest-api/endpoints#delete-a-project

There is also a `vercel project rm` command to delete projects via the CLI. You can find more info about that at https://vercel.com/docs/cli/project

**Reply by kashyapro** [reply]
Thanks @amyegan for the help.
